src/RequestProcessor.cs

```diff
@@ -97,6 +97,7 @@ internal StreamingMessage ProcessWorkerInitRequest(StreamingMessage request)
         {
             var workerInitRequest = request.WorkerInitRequest;
             Environment.SetEnvironmentVariable("AZUREPS_HOST_ENVIRONMENT", $"AzureFunctions/{workerInitRequest.HostVersion}");
+            Environment.SetEnvironmentVariable("POWERSHELL_DISTRIBUTION_CHANNEL", $"Azure-Functions-{workerInitRequest.HostVersion}");
```

Comment:
I think a colon works better as a separator as that's what we use in our MSI

```suggestion
            Environment.SetEnvironmentVariable("POWERSHELL_DISTRIBUTION_CHANNEL", $"Azure-Functions:{workerInitRequest.HostVersion}");
```

Reply:
Sounds good. Code updated.